### DISCUSSION TITLE
Welcome to Laboratory Discussions!

### DISCUSSION THREAD

**Discussion by jirhiker** [NMGRL/Laboratory#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mjz1983** [comment]
Matt Zimmerer - Bureau field geologist and frequent user of the NMGRL. Most research focuses on volcanic/magmatic systems.
